scripts/run_server_tests.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
```

Comment:
This script is not yet called during the testing process.

We thought it would be useful to have a real Postgres client (psql in this case) connect to the server and execute some queries as a form of integration test.

This could run as a Jenkins build stage, simlilar to the `run_system_test.sh` script. However, we'd require the Postgres client to be installed on the build server first.

Should we take care of that in the scope of this PR?

Reply:
No, I’ll open an issue